Open Judge System/Workers/OJS.Workers.Executors/RestrictedProcessExecutor.cs

```diff
@@ -167,6 +167,12 @@ public RestrictedProcessExecutor(int baseTimeUsed, int baseMemoryUsed)
 
                 // Report exit code and total process working time
                 result.ExitCode = restrictedProcess.ExitCode;
+
+                if (result.ExitCode != 0 && result.ExitCode != -1)
```

Comment:
Why not include this check in the `if` statement that checks if the `result.ErrorOutput` `IsNullOrEmpty` and also makes the type `RunTimeError` and maybe make it the last check in the executor, to be sure that no other result will override the `RunTimeError` output